include/wx/msw/ole/oleutils.h

```diff
@@ -66,18 +66,45 @@ inline void wxOleUninitialize()
 class WXDLLIMPEXP_CORE wxBasicString
 {
 public:
-    // ctors & dtor
-    wxBasicString(const wxString& str);
-    wxBasicString(const wxBasicString& bstr);
-    ~wxBasicString();
 
-    wxBasicString& operator=(const wxBasicString& bstr);
-
-    // accessors
-        // just get the string
+	// Takes over the ownership of bstr
+    wxBasicString(BSTR bstr = NULL);
```

Comment:
OK, I will add `explicit` to the current ctor taking a `BSTR`.

As for the second paragraph, there already is [`wxBasicString& wxBasicString::operator=(BSTR bstr)`](https://github.com/wxWidgets/wxWidgets/pull/502/files#diff-f29bb280f7a255f709351eb4fde3a1bdR113) which takes over the ownership?